gulpfile.js

```diff
@@ -13,22 +13,48 @@
 const compilerPackage = require('google-closure-compiler');
 const gulp = require('gulp');
 const sourcemaps = require('gulp-sourcemaps');
+const rollup = require('rollup-stream');
+const source = require('vinyl-source-stream');
 
 const closureCompiler = compilerPackage.gulp();
 
-gulp.task('default', () => {
-  return gulp.src('./src/custom-elements.js', {base: './'})
+function createDefaultTask(closureOptions = {}) {
+  return function() {
+    const defaultClosureOptions = {
+      compilation_level: 'ADVANCED',
+      warning_level: 'VERBOSE',
+      language_in: 'ECMASCRIPT6_STRICT',
+      language_out: 'ECMASCRIPT5_STRICT',
+      externs: ['externs/html5.js', 'externs/custom-elements.js'],
+      dependency_mode: 'STRICT',
+      entry_point: ['/src/custom-elements'],
+      js_output_file: 'custom-elements.min.js',
+      new_type_inf: true,
+      rewrite_polyfills: false,
+    };
+
+    return gulp.src('./src/*.js', {base: './'})
       .pipe(sourcemaps.init())
-      .pipe(closureCompiler({
-          compilation_level: 'ADVANCED',
-          warning_level: 'VERBOSE',
-          language_in: 'ECMASCRIPT6_STRICT',
-          language_out: 'ECMASCRIPT5_STRICT',
-          externs: ['externs/html5.js', 'externs/custom-elements.js'],
-          js_output_file: 'custom-elements.min.js',
-          new_type_inf: true,
-          rewrite_polyfills: false,
-        }))
+      .pipe(closureCompiler(Object.assign(defaultClosureOptions, closureOptions)))
       .pipe(sourcemaps.write('/'))
       .pipe(gulp.dest('./'));
+  }
+}
+
+gulp.task('default', createDefaultTask());
+gulp.task('debug', createDefaultTask({
+  compilation_level: 'SIMPLE',
```

Comment:
The idea was to have `debug` be the same as `default` without name minification (but with renaming) and `rollup` be just enough to de-module-ify everything and change as little else as possible. Maybe `rollup` should become `debug` and this one should be removed?